nomad/structs/structs.go

```diff
@@ -822,8 +822,10 @@ type EvalUpdateRequest struct {
 	WriteRequest
 }
 
-// EvalDeleteRequest is used for deleting an evaluation.
-type EvalDeleteRequest struct {
+// EvalReapRequest is used for reaping evaluations and allocation. This action
+// is performed by the core scheduler which is periodic, or manually by
+// operators using the eval delete RPC.
```

Comment:
Is it the case an `EvalReapRequest` can be manually triggered?

Reply:
The comment is because the reap and delete RPCs ultimately use the same encapsulation struct to delete evals via Raft. This avoids the need for new messages. I'll try and clear up the comment to make this clearer.